Sources/Optimizely+Decide/OptimizelyUserContext.swift

```diff
@@ -122,15 +135,105 @@ public class OptimizelyUserContext {
         guard let optimizely = self.optimizely, optimizely.config != nil else {
             throw OptimizelyError.sdkNotReady
         }
-
+        
         try optimizely.track(eventKey: eventKey,
                              userId: userId,
                              attributes: attributes,
                              eventTags: eventTags)
     }
+    
+}
 
+// MARK: - ForcedDecisions
+
+extension OptimizelyUserContext {
+    
+    public func setForcedDecision(flagKey: String,
+                                  ruleKey: String? = nil,
```

Comment:
`ruleKey` can we add comments, that it is for both experimentRule or deliveryRule.